Amazon.js

```diff
@@ -354,12 +348,23 @@ function scrape(doc, url) {
 			}
 		}
 	}
-
+	if (!els.length) {
```

Comment:
Because movies still use an information table that will match the expression. However this is just a subset of the information and we want to extract from the detailBullets_feature_div for movies as well since this contains all the information elements we want to capture.